### PR TITLE
Add Group Model

### PR DESCRIPTION
**Pull request by JuanequeX**
# Description
 - This configutation added and integrated the group model to made a sctructure to test the data that we requiered to development the controller by groups.

 # Testing
 - Can you testing it run the rspec  models tests in teh file `./spec/model/group_spec.rb`

### CODE REVIEW

**Review by CUBOFIG — APPROVED** [review]
LGTM! 🕷 🕷 🕷

**Review by xFerchoVJ — APPROVED** [review]
lgtm

**Review by YilKanda — APPROVED** [review]
LGTM!

**Comment by JuanequeX** [comment]
Sorry, i need to modify in my db files the kind of data to amount and delete the spent_type because we don't nedeed in our project

**Review by YilKanda — DISMISSED** [review]

File: db/migrate/20211117234907_remove_spent_type_from_groups.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class RemoveSpentTypeFromGroups < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :groups, :spent_type, :boolean
+  end
+end
```
Comment (EduardoGHdez) [inline db/migrate/20211117234907_remove_spent_type_from_groups.rb:5]: Is this migration needed?

I don't see the `spent_type` added before

File: db/migrate/20211118000407_change_amount_to_be_decimal_in_groups.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ChangeAmountToBeDecimalInGroups < ActiveRecord::Migration[6.1]
+  def change
+    change_column :groups, :amount, :decimal
```
Comment (EduardoGHdez) [inline db/migrate/20211118000407_change_amount_to_be_decimal_in_groups.rb:3]: I see the initial migration is taking care of this

Is this migration needed?

File: db/migrate/20211117234907_remove_spent_type_from_groups.rb (hunk 1 above)
Comment (JuanequeX) [reply db/migrate/20211117234907_remove_spent_type_from_groups.rb:5]: We created the model but after to created it we analyze that it are not necessary for our logic, i forgot to remove this migration, i remove it right now, thanks

Comment (JuanequeX) [reply db/migrate/20211117234907_remove_spent_type_from_groups.rb]: Done @EduardoGHdez

File: db/migrate/20211118000407_change_amount_to_be_decimal_in_groups.rb (hunk 2 above)
Comment (JuanequeX) [reply db/migrate/20211118000407_change_amount_to_be_decimal_in_groups.rb:3]: we created this migration to change the value: integer of amount to decimal, if we are solved this change in local we should are not push this migration?

Comment (JuanequeX) [reply db/migrate/20211118000407_change_amount_to_be_decimal_in_groups.rb]: i solve it

DONE

**Review by jestradadeveloper — APPROVED** [review]
LGTM

**Review by CUBOFIG — APPROVED** [review]
LGTM

**Review by Ilyeo — APPROVED** [review]

**Pull request merged**